docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -276,6 +276,14 @@ to be configured individually using the following environment variables.
 If you want to use outbound Connectors for a single tenant that is different
 from the default tenant, you can specify a different default tenant ID using:
 
+It is possible to adjust the polling interval of connectors polling process definitions to Operate by setting the environment variable `CAMUNDA_CONNECTOR_POLLING_INTERVAL`. This variable allows you to control how often Connectors fetch the process definitions, with the interval specified in milliseconds. For example, setting `CAMUNDA_CONNECTOR_POLLING_INTERVAL=20000` will configure the Connectors to poll every 20 seconds.
```

Comment:
Non-blocking, but I've been seeing some inconsistency between connectors and Connectors.

I don't think the docs team has yet gone through and lowercased connectors yet, and until we do so, I would recommend upper casing for consistency.

CC @mesellings